gotool/gotool_test.go

```diff
@@ -0,0 +1,74 @@
+package gotool
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type TestCommander struct {
+	ExecuteCommandFn func(command string, args ...string) (string, error)
+}
+
+func (c TestCommander) ExecuteCommand(command string, args ...string) (string, error) {
+	if c.ExecuteCommandFn == nil {
+		panic("You have to override TestCommander.ExecuteCommand function in tests")
+	}
+	return c.ExecuteCommandFn(command, args...)
+}
+
+func Test_ListPackages(t *testing.T) {
+	testCases := []struct {
```

Comment:
This method is a bit bloaty and hard to read. I suggest moving out the test cases definition to clear it up